cmd/openshift-install/create.go

```diff
@@ -468,7 +468,7 @@ func logComplete(directory, consoleURL string) error {
 	logrus.Info("Install complete!")
 	logrus.Infof("To access the cluster as the system:admin user when using 'oc', run 'export KUBECONFIG=%s'", kubeconfig)
 	logrus.Infof("Access the OpenShift web-console here: %s", consoleURL)
-	logrus.Infof("Login to the console with user: kubeadmin, password: %s", pw)
+	logrus.Infof("Login to the console with user: 'kubeadmin' password: '%s'", pw)
```

Comment:
maybe use `%q`

also  i think we proabably still `, and`

Reply:
ping @pdvian

Reply:
Ack. Let me do the changes.